### DISCUSSION TITLE
nerdctl cp is an alternative to nerdctl export?

### DISCUSSION THREAD

**Discussion by Pymann** [containerd/nerdctl#2570] [Q&A]
Can you use "nerdctl cp" for a started and stopped image instead of the "nerdctl export" command?

**Comment by fahedouch** [comment]
you can use `nerdctl cp` for started and stopped  containers

**Reply by Pymann** [reply]
and can you extract complete filesystem?

**Reply by fahedouch** [reply]
No, this should be done by export, which is not yet implemented.
